.github/workflows/pinot_tests.yml

```diff
@@ -133,7 +133,7 @@ jobs:
           MAVEN_OPTS: >
             -Xmx2G -DskipShade -DfailIfNoTests=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=25
             -Dmaven.wagon.http.retryHandler.count=30 -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false
-            -B -ntp
+            -B -ntp -P codecoverage
```

Comment:
I think these changes should be applied in https://github.com/apache/pinot/blob/5d58102d0ac7366d578ea1012ec125379ef70e7c/.github/workflows/scripts/pr-tests instead

Reply:
In that folder there are 4 scripts. Each one call maven at least once. There is where we have been setting the profiles.

I don't have a strong opinion between applying the profile in these scripts or here.

Reply:
But after this change it looks like all executions are failing because `-B is not recognized as JVM option`